src/components/widget/d2l-teacher-course-creation-success.js

```diff
@@ -28,13 +61,64 @@ class TeacherCourseCreationSuccess extends BaseMixin(LitElement) {
 
 	connectedCallback() {
 		super.connectedCallback();
+
+		if (this.pageData && 'generatedOrgUnitId' in this.pageData) {
+			this.generatedOrgUnitId = this.pageData.generatedOrgUnitId;
+		}
+	}
+
+	_donePressed() {
+		this.changePage(PAGES.WELCOME_PAGE);
+	}
+
+	_anotherCoursePressed() {
+		this.changePage(PAGES.INPUT_PAGE);
+	}
+
+	_getCourseEnrollHref(orgUnitId) {
+		return '/d2l/lms/classlist/classlist.d2l?ou='.concat(orgUnitId);
```

Comment:
Can we use string interpolation?
```suggestion
		return `/d2l/lms/classlist/classlist.d2l?ou=${orgUnitId}`;
```